### DISCUSSION TITLE
Chip Status "Not Connected"

### DISCUSSION THREAD

**Discussion by usekevin1988** [tbnobody/OpenDTU#359] [Q&A]
Hallo zusammen. Wenn ich die DTU in die Steckdose Stecke, zeigt mir das Webinterface kurz an, dass das NRF Modul Connected ist, und auch, dass es das Plus Modul ist. Nach wirklich kurzer Zeit (noch bevor eine Verbindung zum WR Aufgebaut wurde) Ändert sich die Anzeige zu "Not Connected".

Mit AhoyDTU hat es 1A funktioniert. (selbe Hardware)

Verkabelung habe ich jetzt mehrfach geprüft. an dem NRF  ist ein Kondensator angelötet.

Jemand ne Idee?

Ich glaub es liegt an meinem NRF, weil es geht auch mit ahoy nicht mehr... hab mir mal ein neues bestellt, und werde berichten

**Comment by raccoon0815** [comment]
Hallo @usekevin1988 
 ja die Qualität der NRF24L01+ ist nicht immer berauschend. Ich bin aktuell immer gut mit den NRF24L01+ Modulen von DollaTek   zurecht gekommen. Kein einziges ist bisher ausgefallen. Bei den Kuman nRF24L01+ habe ich alle wieder zurück gesendet, da hier jedes zweite Modul defekt war.
Die besten Erfahrungen (Empfang und Kompatibilität bei PoE) habe ich mit den BYTE NRF24L01+ E01-ML01DP5 Modulen gemacht.

**Comment by ghost** [comment]
Ich habe [diese Module von makershop.de](https://www.ebay.de/itm/255283194279) verwendet und kann sie empfehlen. (Sind allerdings momentan dort leider nicht verfügbar.)

Damit hatte ich auf Anhieb stabile Verbindung zum HM-600, auch über größere Entfernungen (z.B. 30m mit Mauer oder Blechdach dazwischen). Es arbeitet jetzt so seit geraumer Zeit absolut zuverlässig und flüssig. Die Default-Einstellung für PA Level ("Minimum") habe ich dafür nicht ändern müssen und empfehle es auch nicht.
